tests/agent_equipment/test_resolver.py

```
@@ -0,0 +1,2467 @@
+            for observation in inventory.observations
+            if observation.harness == group.harness
+            and observation.route_identity == group.route_identity
+        )
+
+        self.assertEqual(
+            _observation_binding_diagnostics(group, selected, observation),
+            (),
+        )
+
+        mutable = observation.document.__class__(observation.document._items)
+        changed = {key: value for key, value in mutable.items()}
```

Ruff (0.16.1)

[warning] Unnecessary dict comprehension (rewrite using `dict()`)

Rewrite using `dict()`

(C416)